inc/nux/class-storefront-nux-starter-content.php

```diff
@@ -997,7 +997,7 @@ private function _starter_content_products() {
 				$products[ $symbol ]['post_name'] = $symbol;
 			}
 
-			return apply_filters( 'storefront_starter_content_products', $products );
+			return apply_filters( 'storefrontstarter_content_products', $products );
```

Comment:
We can't change the name of this filter without good reason - this will break any sites that implement the filter.

Reply:
Similar to the other case, I cannot explain what happened here. Of course, changing the name of a filter does not make any sense. Not sure if this change got triggered by linting or something else. It was definitely not consciously and I would have removed it from the push if I had caught it. That said, I also reverted this change and pushed an updated version.